### ISSUE TITLE
Problem with SQS message attributes

### ISSUE THREAD

**Issue by itaikeren** [electricimp/AWSSQS#4]
Sending a message with the SQS library is working fine until I try to add message attributes (last 3 lines).
This is my code:

```js
// SQS send message parameters
local sendParams = {
"QueueUrl" : "https://sqs.us-east-2.amazonaws.com/754....",
"MessageGroupId" : format("%s",theDeviceID),
"MessageBody" : Data,
"MessageDeduplicationId" : id,

"MessageAttribute.1.Name" : "test",
"MessageAttribute.1.Value" : "testmessage",
"MessageAttribute.1.Type" : "String",

};
```

To make sure AWS is indeed looking for the message attribute I send (in purpose) only the 'Name' of the message attribute (without the 'Value' and 'Type') and got that error message:

`The message (user) attribute 'test' must contain a non-empty message attribute value.`

This means AWS understands what is "MessageAttribute.1.Name", and looking for its value.
When I add the 'Value' and 'Type' I got that error message:

`Unexpected complex element termination`

I was following according to what is written in the docs: [Imp API - AWS Simple Queue Service](https://developer.electricimp.com/libraries/webservices/awssqs)

**Comment by itaikeren** [comment]
I create a PR with a fix for this issue: #5

**Comment by smittytone** [comment]
Thanks for the PR!
